reference/user_defined_functions.rst

```diff
@@ -330,6 +330,13 @@ Arguments
 
 **node_port:** The port on which PostgreSQL is listening on the worker node.
 
+**group_id:** A group of one primary server and zero or more secondary
+servers, relevant only for streaming replication.  Default 0
+
+**node_role:** Whether it is 'primary' or 'secondary'. Default 'primary'
+
+**node_cluster:** The sub-64-character cluster name. Default 'default'
```

Comment:
I think we can leave it out, as we don't specify it for other text types. The code should be robust to error meaningfully in case someone exceeds such boundaries.